### PR TITLE
fix some typoes

### LINKED ISSUE #1564: Typo?
inadvernetly 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a47bf610-692d-9d40-d85f-aec1e2a87a07
* Version Independent ID: 3c94634a-4b17-bc13-7436-0c61aea03ab6
* Content: [Configuring a DbContext - EF Core](https://docs.microsoft.com/en-us/ef/core/miscellaneous/configuring-dbcontext#feedback)
* Content Source: [entity-framework/core/miscellaneous/configuring-dbcontext.md](https://github.com/aspnet/EntityFramework.Docs/blob/master/entity-framework/core/miscellaneous/configuring-dbcontext.md)
* Product: **entity-framework**
* Technology: **entity-framework-core**
* GitHub Login: @rowanmiller
* Microsoft Alias: **divega**

### PR DESCRIPTION
**Pull request by SimonCropp**
also fixes #1564

### CODE REVIEW

File: entity-framework/core/querying/filters.md [hunk 1]
```diff
@@ -25,20 +25,20 @@ First, define the entities:
 
 [!code-csharp[Main](../../../samples/core/QueryFilters/Program.cs#Entities)]
 
-Note the declaration of a __tenantId_ field on the _Blog_ entity. This will be used to associate each Blog instance with a specific tenant. Also defined is an _IsDeleted_ property on the _Post_ entity type. This is used to keep track of whether a _Post_ instance has been "soft-deleted". That is, the instance is marked as deleted without physically removing the underlying data.
```
Comment (ajcvickers) [inline entity-framework/core/querying/filters.md]: Isn't "_tendantId" correct here?

Comment (SimonCropp) [reply entity-framework/core/querying/filters.md]: before my change it had two `_` which resulted in the entire line being bold

**Comment by divega** [comment]
Thanks @SimonCropp!

**Comment by SimonCropp** [comment]
@divega thank you for your work on EF :)

**Pull request merged**